data-raw/process_data_sets_anna.R

```diff
@@ -0,0 +1,20 @@
+#load packages for taming
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+# foul-balls -------------------------------------------------------------------
+foul_balls <- read_csv("data-raw/foul-balls/foul-balls.csv") %>% 
+  clean_names() %>% 
+  mutate(
+    type_of_hit = as.factor(type_of_hit),
+    camera_zone = as.factor(camera_zone),
+    used_zone = as.factor(used_zone),
+    predicted_zone = as.factor(predicted_zone)
+  )
```

Comment:
The following three matchups have `VS` instead of `vs`:
```
"Seattle Mariners VS Minnesota Twins"         
"Baltimore Orioles VS Minnesota Twins"        
"Pittsburgh Pirates VS Milwaukee Brewers" 
```
the other 7 matchups have `vs`.

Furthemore, `"Los Angeles Dodgers vs Arizona Diamondsbacks"` has a typo in "Diamondbacks".

Even though these two errors are not on you, but rather on 538, could you indulge my extreme pedantry and do a `str_replace_all()` within the above mutate and fix these typos?

Reply:
In fact, if you wanted to go whole hog after you're done here, you could make a PR to the original data on 538's github: https://github.com/fivethirtyeight/data/tree/master/foul-balls